ci/run_test.sh

```diff
@@ -9,6 +9,16 @@ if [[ $(( 1 % ${CI_NODE_TOTAL} )) -eq ${CI_NODE_INDEX} ]]; then
     "${BASEDIR}/validate.sh"
 fi
 
-"${BASEDIR}/run_test_h2.sh"
-"${BASEDIR}/run_test_pg.sh"
+WITH_BUILD_UI=""
+WITHOUT_BUILD_UI="-x buildUi"
+
+# Don't need to build UI always
+if [[ $(( 2 % ${CI_NODE_TOTAL} )) -eq ${CI_NODE_INDEX} ]]; then
+  BUILD_UI_OPT=${WITH_BUILD_UI}
+else
+  BUILD_UI_OPT=${WITHOUT_BUILD_UI}
+fi
+
+"${BASEDIR}/run_test_h2.sh" ${WITHOUT_BUILD_UI}
```

Comment:
```suggestion
"${BASEDIR}/run_test_h2.sh" ${BUILD_UI_OPT}
```

Isn't this line `${BUILD_UI_OPT}` instead of `${WITHOUT_BUILD_UI}`?

Reply:
Ah, I think it's enough to build UI either with H2 or PostgreSQL not with the both.

Reply:
I see, then please ignore my comment. Thank you for your explanation.

Reply:
I'll add a comment to clarify my intention :)